zap/src/main/java/org/zaproxy/zap/extension/brk/BreakPanel.java

```diff
@@ -541,6 +570,10 @@ public BreakButtonsUI(String name, BreakPanelToolbarFactory breakToolbarFactory)
             toolBar.add(breakToolbarFactory.getBtnContinue());
             toolBar.add(breakToolbarFactory.getBtnDrop());
             toolBar.add(breakToolbarFactory.getBtnBreakPoint());
+            toolBar.add(brkOnJsButton);
```

Comment:
Might worth adding a separator before this one and before the scope button, as they have opposite behaviour.

Reply:
Yes it makes sense